app/src/main/java/org/mozilla/focus/activity/MainActivity.kt

```diff
@@ -42,6 +44,10 @@ open class MainActivity : LocaleAwareAppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        if (MvpFeatureManager.isEnabled) {
```

Comment:
Let's rebase this PR since you mentioned we don't need this any more.

Reply:
From what I noticed on [this pr](https://github.com/mozilla-mobile/focus-android/pull/4956) I'm still constrained to keep this to load dark theme resources when FeatureFlags.isMvp= true